### DISCUSSION TITLE
How to get Importer and Gocardless to not authenticate with the bank every time I want to refresh

### DISCUSSION THREAD

**Discussion by cydnax** [firefly-iii/firefly-iii#8698] [Firefly III] [answered]
Every time I want to refresh my transactions via gocardless I have to enter my id and key and then authenticate with my bank. Is this normal behavior or have i done something wrong? 

In Actual, i set it up once and then  just refresh and new transactions are added.

**Comment by JC5** [comment, marked as answer]
You can download the configuration file, and use it again for the next time you want import data.

**Reply by cydnax** [reply]
Thanks. Working now.
